firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -32,8 +32,8 @@ class ActivityStreamTest: BaseTestCase {
                                LaunchArguments.SkipContextualHints,
                                LaunchArguments.DisableAnimations]
         }
-        launchArguments.append(LaunchArguments.SkipAddingGoogleTopSite)
-        launchArguments.append(LaunchArguments.SkipSponsoredShortcuts)
+        // launchArguments.append(LaunchArguments.SkipAddingGoogleTopSite)
+        // launchArguments.append(LaunchArguments.SkipSponsoredShortcuts)
```

Comment:
@cyndichin Could you please confirm the behaviour of these two `LaunchArguments`? These `LaunchArguments` removes all top sites and sponsored shortcuts.

Reply:
The behavior should remain the same, I'll create a follow up investigation ticket. I'm at a work week this week, but will try to investigate during free time. Thanks!